### PR TITLE
chore: raise if several interfaces are found

### PR DESCRIPTION
**Pull request by ThibaultFy**

### CODE REVIEW

**Review by SdgJlbl — APPROVED** [review]
Thanks for taking care of that well-hidden source of bug 🕵️

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Review by samlesu — APPROVED** [review]
LGTM

File: substratools/utils.py [hunk 1]
```diff
@@ -101,9 +101,17 @@ def load_interface_from_module(module_name, interface_class, interface_signature
...
-        if issubclass(obj, interface_class):
-            return obj()  # return interface instance
+        if issubclass(obj, interface_class) and obj != interface_class:
+            found_interfaces.append(obj)
+
+    if len(found_interfaces) == 1:
+        return found_interfaces[0]()  # return interface instance
+    elif len(found_interfaces) > 1:
+        raise exceptions.InvalidInterfaceError(
+            f"Multiple interfaces found in module '{module_name}': {found_interfaces}"
```
Comment (jeandut) [inline substratools/utils.py:113]: Change the error message no one knows what interface mean

Comment (jeandut) [reply substratools/utils.py:113]: "Multiple opener sub-classes defined in the `opener.py` file, this is not allowed."

Comment (ThibaultFy) [reply substratools/utils.py:113]: The opener and the different subclasses will appear in `module_name` and `found_interfaces`

Comment (jeandut) [reply substratools/utils.py:113]: I disagree you don't have the info of interface_class which is in this case opener

Comment (jeandut) [reply substratools/utils.py:113]: Please add at least interface_class to the debug print statement

**Pull request merged**